src/script/keyboardevents.ts

```diff
@@ -0,0 +1,21 @@
+export function handleKeyPress(this: any, event: KeyboardEvent): void {
+    const key: string = event.key;
```

Comment:
Here we can provide one early return statement if key is not found

```suggestion
    const key: string = event.key;
    
    if(!key) return;
```